### ISSUE TITLE
[HTML] Experience is logically a table

### ISSUE THREAD

**Issue by Idrinth** [SPIRITORBS/portfolio-HTML-CSS-JS#5]
Currently you are using headers and article to build a table like structure for your experience listing. This should be replaced by tabel, table row and table cell to make it more semantic and allow screenreaders and crawlers to understand the context more easily.
